### ISSUE TITLE
Package not installable via pip3

### ISSUE THREAD

**Issue by soerenschneider** [ZivaVatra/flac2all#42]
The current flac2all package is not installable via pip3 and yields the following output:

```
$ pip3 install --user flac2all
Collecting flac2all
  Using cached https://files.pythonhosted.org/packages/93/9e/246405a84106f7d5073388d83c3cf65e95298379c56082ec181a5c18313e/flac2all-4.3.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-build-rm5nnfdn/flac2all/setup.py", line 17, in <module>
        with open("./flac2all_pkg/version", 'r') as fd:
    FileNotFoundError: [Errno 2] No such file or directory: './flac2all_pkg/version'

    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-build-rm5nnfdn/flac2all/
```

**Comment by ZivaVatra** [comment]
Yes :-(

Unfortunately I did not realise python3 changed the behaviour with relative imports (used to be implicit, but now needs to be explicit). As a result while my code worked fine in my dev environment, the pip package breaks as it can't find the modules. 

As flac2all make huge use of relative importing, this is taking a while to debug and test. I hope to have the pip package up and running in the next few days.

**Comment by ZivaVatra** [comment]
Could you please try to install the test pip version somewhere, to make sure it works for you? 

You can do it for a non root user like so: `pip3 install --user -i https://test.pypi.org/simple/ --no-cache-dir  flac2all`

**Comment by soerenschneider** [comment]
I can confirm it's working again. Great job, thanks!

```
[vagrant@localhost ~]$ pip3 install --user -i https://test.pypi.org/simple/ --no-cache-dir flac2all
Collecting flac2all
  Downloading https://test-files.pythonhosted.org/packages/8f/5d/2719353def592516dd198a758d0ca60a3ec416e89ab7a447e15b0bf01637/flac2all-5.4-py3-none-any.whl
Installing collected packages: flac2all
Successfully installed flac2all-5.4
[vagrant@localhost ~]$ flac2all -h
Usage:
Flac2all python script, DEVELOPMENT. Copyright 2003 ziva-vatra
Licensed under the GPLv3 or later. Please see included LICENCE file,
or visit http://www.gnu.org/licenses/gpl.txt for the full licence.

Dev website: https://github.com/ZivaVatra/flac2all

	Usage: /home/vagrant/.local/bin/flac2all enctype1[,enctype2..] [options]  inputdir

	Valid encode types are as follows:
		mp3
		vorbis
		aacplus
		opus
		flac
		test
	You can specify multiple encode targets with a comma seperated list.



Options:
  -h, --help            show this help message and exit
  -c, --copy            Copy non flac files across (default=False)
  -C, --curses          Use curses UI
  --ffmpeg-options=FFMPEGOPTS
                        Comma delimited options to pass to ffmpeg. Exact
                        options will vary based on which of the ffmpeg codecs
                        you are using
  --opus-options=OPUSENCOPTS
                        Colon delimited options to pass to opusenc.        Any
                        oggenc long option (one with two '--' in front) can be
                        specified        in the above format.
  --vorbis-options=OGGENCOPTS
                        Colon delimited options to pass to oggenc,for
                        example: 'quality=5:resample
                        32000:downmix:bitrate_average=96'.        Any oggenc
                        long option (one with two '--' in front) can be
                        specified        in the above format.
  --lame-options=LAMEOPTS
                        Options to pass to lame,for example:
                        '-preset extreme:q 0:h:-abr'. Any lameoption can be
                        specified here, if you want a short option (e.g.
                        -h),then just do 'h'. If you want a long option (e.g.
                        '--abr'), then you needa dash: '-abr'
  -o DIR, --outdir=DIR  Set custom output directory (default='./')
  -f, --force           Force overwrite of existing files (by default we skip)
  -t THREADS, --threads=THREADS
                        How many threads to run in parallel (default:
                        autodetect    [found 1 cpu(s)] )
  -n, --nodirs          Don't create Directories, put everything together
  -m, --master          Start flac2all in master mode (for clustering).
```

**Comment by ZivaVatra** [comment]
Glad to hear it! Changes have been merged and pushed to v5.1, and new pip package deployed, so you should be able to upgrade via pip as normal to fix the issue.
